lab_1_classify_by_unigrams/main.py

```diff
@@ -20,6 +22,13 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned
     """
+    if not isinstance(text, str):
+        return None
+    list_of_letters = []
+    for character in text:
```

Comment:
better to call `.lower()` for full string

Reply:
done